### PR TITLE
feat: add Select style

### PR DESCRIPTION
**Pull request by GoOz**
- [x] Added `<select>` style
- [x] Added autoprefixer for CSS building (KSS and standalone UI)
- [x] Added some missing arrow icons

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,13 +23,14 @@
     "deploy:doc": "git-directory-deploy --directory build/ --branch gh-pages",
     "prebuild:css": "mkdir dist/",
     "prebuild:css:app": "mkdir -p build/styleguide",
-    "build:css": "stylus -c --include stylus -o dist/cozy-ui.min.css stylus/cozy-ui/build.styl",
-    "build:css:app": "stylus --include stylus -o build/styleguide/app.css stylus/cozy-ui/build.styl",
+    "build:css": "stylus -c --include stylus -o dist/cozy-ui.min.css stylus/cozy-ui/build.styl --use ./node_modules/autoprefixer-stylus --with \"{ browsers: ['last 2 versions'] }\"",
+    "build:css:app": "stylus --include stylus -o build/styleguide/app.css stylus/cozy-ui/build.styl --use ./node_modules/autoprefixer-stylus --with \"{ browsers: ['last 2 versions'] }\"",
```
Comment (ptbrowne) [inline package.json]: At this point, it may be more readable to use the JS API instead of the CLI -> move the logic to a dedicated script ?

**Comment by GoOz** [comment]
@ptbrowne Done! Sorry about the omission.
https://gooz.github.io/cozy-ui/styleguide/section-components.html

File: stylus/cozy-ui/build.styl [hunk 2]
```diff
@@ -361,6 +361,31 @@
...
+            <option value="" disabled selected>Choose your option</option>
+            <option value="1">Option 1</option>
+            <option value="2">Option 2</option>
+            <option value="3">Option 3</option>
+            <option value="4">Option 4</option>
+        </select>
+    </div>
+ </form>
+
+ .is-error - Error variant
```
Comment (ptbrowne) [inline stylus/cozy-ui/build.styl]: Shouldn't this be a modifier class like .c-select--error ?

Comment (GoOz) [reply stylus/cozy-ui/build.styl]: Well I used the *SuitCSS* approach on this and prefered its [naming convention](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md), I used it a lot before but I may have forgot to update the Doc, sorry 😕. 
It's a lot like BEM actually but the main difference is it's seperating variant classes from state classes. 
I used this mainly for error and active states so far.

**Comment by GoOz** [comment]
[Basic Select style](https://trello.com/c/xPhyVmAx/120-basic-select-style)

**Comment by ptbrowne** [comment]
LGTM

File: stylus/cozy-ui/build.styl (hunk 2 above)
Comment (GoOz) [reply stylus/cozy-ui/build.styl]: That's the old style (keeping it for retrocompatibility) everything below [this line](https://github.com/GoOz/cozy-ui/blob/aa81512afe6c2752e7418ca646a07df1c0ce79cf/stylus/components/forms.styl#L229) is new 
I'm still refactoring bit by bit but the old style won't stay.

**Pull request merged**